changelog.d/11685.misc

```diff
@@ -0,0 +1,2 @@
+Run `pyupgrade --py37-plus --keep-percent-format` on Synapse.
+
```

Comment:
Seems to have an extra blank line, not sure if the news scripts will deal with that OK?

```suggestion
Run `pyupgrade --py37-plus --keep-percent-format` on Synapse.
```